nilearn/decoding/tests/test_decoder.py

```diff
@@ -94,50 +95,53 @@ def test_BaseDecoder_check_estimator():
     # if it is a string, and if not, then raise the error
```

Comment:
Write this as a docstring.

Reply:
`...one of the supported estimators or a string...`

Rename the test function to `test_BaseDecoder_check_estimator_validity`